src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown

```diff
@@ -112,19 +118,20 @@ The `package.json` file for the project looks like this:
 }
 {% endhighlight %}
 
-## Announcing the web site
+## Announcing the website
 
-The latest version (M49) of Chrome on Android supports Physical Web, which
+The latest version (M49) of Chrome&reg; on Android&#42; supports Physical Web, which
```

Comment:
Change to "Beginning with version 49, Chrome on Android supports..." We're two weeks away from 50 going to stable.